modules/kernel/src/main/scala/tofu/syntax/collections.scala

```diff
@@ -0,0 +1,86 @@
+package tofu.syntax
+
+import cats.syntax._
+import cats.{Applicative, Traverse, TraverseFilter}
+import cats.syntax.functor._
+import cats.data.State
+import cats.data.StateT
+import cats.Monad
+import cats.free.Free
+import cats.FlatMap
+object collections
```

Comment:
no newline

Reply:
I suppose it's something scalafmt should watch.
Actually we enabled scalafmt check to avoid any style discussions in reviews, so I purposedly ignore this one